packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -738,7 +738,7 @@ describe('cluster', () => {
     });
 
     // THEN
-    expect(() => cluster.addRotationSingleUser()).toThrow(/without secret/);
+    expect(() => cluster.addRotationSingleUser()).toThrow(/without master secret/);
 
 
```

Comment:
Let's get rid of these empty lines while we're in the area.

Reply:
Ditto.